stepper.c

```diff
@@ -403,7 +402,7 @@ void st_force_check()
   const uint8_t positive_stop = (positive_direction && (FORCE_VAL >= limits.bump_grip_force));
   const uint8_t negative_stop = (!positive_direction && (FORCE_VAL <= limits.bump_grip_force));
     
-  if (positive_stop | negative_stop) {
+  if (positive_stop || negative_stop) {
```

Comment:
I'm fairly confident that a binary works and (in a miniscule way) potentially the more efficient choice here.